src/i18n/locales/pt_BR/translation.json

```diff
@@ -41,31 +56,38 @@
         "actions": {
           "delete_edit_account": "Remover conta inválida e editar novamente"
         }
-      },
-      "heading": "O poder do bitcoin no seu navegador",
-      "title": "O poder do bitcoin no seu navegador"
+      }
     },
     "choose_connector": {
+      "title": {
+        "welcome": "Você já possui uma carteira bitcoin na rede relâmpago?",
+        "options": "Adicionar uma nova conta"
+      },
+      "description": "Você precisa primeiro conectar-se a uma conta ou servidor para poder interagir com seus sites favoritos que aceitam pagamentos em bitcoin!",
       "alby": {
         "title": "Carteira Alby",
+        "description": "Registrar ou acessar conta Alby",
         "pre_connect": {
-          "email_label": "Endereço de E-mail",
+          "title": "Sua Carteira Bitcoin na Alby",
           "login_account": "Registrar ou acessar conta Alby.",
           "host_wallet": "Nós cuidamos da sua carteira Bitcoin!",
-          "title": "Sua Carteira Bitcoin na Alby",
+          "email": {
+            "label": "Endereço de E-mail"
+          },
           "optional_lightning_note": {
             "part1": "Ao registrar uma conta Alby, você pode escolher um",
             "part2": "Endereço Relâmpago",
-            "part4": "saiba mais",
-            "part3": ". É o jeito mais simples para você receber Bitcoins na Rede Relâmpago."
+            "part3": ". É o jeito mais simples para você receber Bitcoins na Rede Relâmpago.",
+            "part4": "saiba mais"
+          },
+          "optional_lightning_address": {
+            "label": "Escolha um Endereço Relâmpago (opcional)",
+            "suffix": "@getalby.com"
           },
-          "optional_lightning_address_suffix": "@getalby.com",
           "errors": {
             "create_wallet_error": "Falha ao criar uma nova carteira"
-          },
-          "optional_lightning_address_label": "Escolha um Endereço Relâmpago (opcional)"
+          }
         },
-        "description": "Registrar ou acessar conta Alby",
         "post_connect": {
```

Comment:
This isn't being used anywhere but has been added from weblate, maybe we should change things there?
//cc @escapedcat 
(Also have to change the order to match English as it would help while comparing - Did it manually in this PR, though, (reg. future translations))